paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -251,11 +251,14 @@ class PaparazziPlugin : Plugin<Project> {
             nativePlatformFileCollection.singleFile.absolutePath
           test.systemProperties["paparazzi.test.record"] = isRecordRun.get()
           test.systemProperties["paparazzi.test.verify"] = isVerifyRun.get()
+          reportOutputDir.get().asFile.deleteRecursively()
         }
 
         test.doLast {
-          val uri = reportOutputDir.get().asFile.toPath().resolve("index.html").toUri()
-          test.logger.log(LIFECYCLE, "See the Paparazzi report at: $uri")
+          val report = reportOutputDir.get().asFile.resolve("index.html")
```

Comment:
It should keep using the path API. Otherwise the link will become unclickable in IDEA.

```
println(file("build.gradle").toURI())
// output: file:/T:/xx/build.gradle
println(file("build.gradle").toPath().toUri())
// output: file:///T:/xx/build.gradle
```

![image](https://github.com/cashapp/paparazzi/assets/2906988/5cd117a0-c4eb-49b8-8442-f8081aeaca14)

Reply:
Ah, great callout. I did not realize it mattered how we got a an `URI` instance. Will fix.